src/ExpandableListItem/ExpandableListItem.js

```diff
@@ -9,18 +9,23 @@ var
 	kind = require('enyo/kind'),
 	Component = require('enyo/Component'),
 	Control = require('enyo/Control'),
+	EnyoHistory = require('enyo/History'),
 	Group = require('enyo/Group');
 
 var
 	Spotlight = require('spotlight');
 
 var
+	HistorySupport = require('../HistorySupport'),
 	Item = require('../Item');
 
 var
 	ExpandableListItemHeader = require('./ExpandableListItemHeader'),
 	ExpandableListItemDrawer = require('./ExpandableListItemDrawer');
 
+var
+	hasHistoryEntry = false; // used to track whether or not we have pushed a history entry for ourselves
```

Comment:
Not sure this should be shared. What was the rationale for it here instead of an instance member?